storage/app/src/main/java/com/google/firebase/quickstart/firebasestorage/java/MainActivity.java

```diff
@@ -193,15 +204,7 @@ private void launchCamera() {
         Log.d(TAG, "launchCamera");
 
         // Pick an image from storage
-        ActivityResultLauncher<String[]> intentLauncher = registerForActivityResult(
-                new ActivityResultContracts.OpenDocument(), fileUri -> {
-                    if (fileUri != null) {
-                        uploadFromUri(fileUri);
-                    } else {
-                        Log.w(TAG, "File URI is null");
-                    }
-                });
-        intentLauncher.launch(new String[]{ "image/*" });
+        this.intentLauncher.launch(new String[]{ "image/*" });
```

Comment:
Same comment as above
```suggestion
       intentLauncher.launch(new String[]{ "image/*" });
```